system/ee/legacy/libraries/channel_entries_parser/components/Custom_field.php

```diff
@@ -104,13 +105,13 @@ public function replace($tagdata, EE_Channel_data_parser $obj, $ft_api)
                     ));
 
                     if (method_exists($obj, $parse_fnc)) {
-                        $entry = $ft_api->apply($parse_fnc, array(
+                        $entry = (string) ft_api->apply($parse_fnc, array(
```

Comment:
@matthewjohns0n looks like you lost the `$` on `$ft_api` here

Reply:
Thanks, fixed!